lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -263,39 +265,31 @@ def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
             'equalyz'   adapt the y and z axes to have equal aspect ratios.
             =========   ==================================================
 
-        adjustable : None or {'box', 'datalim'}, optional
-            If not *None*, this defines which parameter will be adjusted to
-            meet the required aspect. See `.set_adjustable` for further
-            details.
+        adjustable : {'box', 'datalim'}, default: 'box'
+           Defines Which parameter to adjust to meet the aspect ratio.
+
+        - 'box': Change the physical dimensions of the axes bounding box.
+        - 'datalim': Change the x, y, or z data limits.
```

Comment:
```suggestion
            Defines Which parameter to adjust to meet the aspect ratio.

            - 'box': Change the physical dimensions of the axes bounding box.
            - 'datalim': Change the x, y, or z data limits.
```
What I meant is not indented with respect to the adjacent text block. The whole description is of course indented with respect to the parameter name. (and we use 4 spaces for indent there, which is fixed in this suggestion.)